packages/storage/src/chrome/migration.test.ts

```diff
@@ -147,7 +146,7 @@ describe('Storage migrations', () => {
         },
       ]);
 
-      await flushPromises();
+      await Promise.resolve();
```

Comment:
If we remove `flushPromises` we've lost some documentation. Can you add that here?

Reply:
Looks like it was insta merged, but could we still address this one ⬆️

Reply:
it seems unnecessary so i'll just make a PR that removes it.